packages/office-ui-fabric-react/src/components/Nav/Nav.types.ts

```diff
@@ -155,7 +155,8 @@ export interface INavLink {
   icon?: string;
 
   /**
-   * Classname to apply to the icon link.
+   * iconClassName has been deprecated in favor of IIconProps. Use IIconProps.className instead.
```

Comment:
Can you change `IIconProps` (the Type) to `iconProps` (the actual prop of type IIconProps you want to use).

Nitpicky I know, but clarity is everything in the documentation.